hack/make-rules/test.sh

```diff
@@ -32,28 +32,26 @@ export KUBE_CACHE_MUTATION_DETECTOR
 KUBE_PANIC_WATCH_DECODE_ERROR="${KUBE_PANIC_WATCH_DECODE_ERROR:-true}"
 export KUBE_PANIC_WATCH_DECODE_ERROR
 
-kube::test::find_dirs() {
+kube::test::find_go_packages() {
```

Comment:
Do you want to make the same change in 
https://github.com/kubernetes/kubernetes/blob/f0036aac21e03c1c2ec4b8c41283f064f30803c1/hack/make-rules/test-integration.sh#L46-L57
?

In this PR or separately?

Reply:
I think is better to do follow up and get this merged

Reply:
Agreed.

Reply:
Agree

Reply:
💯, let's merge and iterate